server/control/sessionControl.js

```diff
@@ -55,20 +59,23 @@ class SessionController {
   static reviewMentor(req, res) {
     const { score, remark } = req.body;
     const isSession = sessions.find(f => f.sessionId == req.params.id);

```

Comment:
We talked about naming variables with more meaninful names not `f` or `p`

Reply:
yes, I was going to do that on a separate branch along with the dry testing issue or should I just do it on this branch and commit again